src/crimson/os/seastore/transaction_manager.h

```diff
@@ -1023,7 +1024,8 @@ class TransactionManager : public ExtentCallbackInterface {
 	remap_laddr,
 	remap_length,
 	intermediate_key,
-	intermediate_base);
+	intermediate_base,
+	false);
```

Comment:
We have already specialized `lba_manager->decref_extent()` with bool `cascade_remove`, is it better to fix that instead of specializing both places?

Reply:
> We have already specialized `lba_manager->decref_extent()` with bool `cascade_remove`, is it better to fix that instead of specializing both places?

I think specializing `lba_manager->decref_extent()` with bool `cascade_remove` can avoid unnecessary lba tree searches, what do you think?

Reply:
Then it might be better to encapsulate a dedicated `lba_manager->remap_mapping_with_clone()` to encapulate all the according specializations (decref+clone+incref) together.

It also make LBAManager interface simpler: `cascade_remove` of `decref_extent()` can be hidden, and `incref_extent(t, addr, delta)` can be dropped from the public interface, does it make sense?

Reply:
will try:-)